tdrs-frontend/src/actions/reports.js

```diff
@@ -55,19 +68,16 @@ export const getAvailableFileList = ({ year, quarter = 'Q1' }) => async (
   }
 }
 
-export const download = ({ year, quarter = 'Q1', section }) => async (
+export const download = ({ id, quarter = 'Q1', section, year }) => async (
   dispatch
 ) => {
   try {
-    if (!year) throw new Error('No year was provided to download action.')
+    if (!id) throw new Error('No id was provided to download action.')
```

Comment:
Is this error something we'd want to log to the server?

Reply:
Yes